meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/kv260/zynqmp-sck-kv-g-rev1.dts

```diff
@@ -39,7 +39,7 @@
 	usbhub: usb5744@2d { /* u43 */
 		compatible = "microchip,usb5744";
 		reg = <0x2d>;
-		reset-gpios = <&gpio 44 GPIO_ACTIVE_HIGH>;
+		reset-gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
```

Comment:
I notice these are under the i2c node.  This driver expects the parent to be a USB node now, if I'm not mistaken?

Reply:
It works correctly with these changes only, so it might be ok.

Will merge as is to unblock this and we can update if needed.

Reply:
> I notice these are under the i2c node. This driver expects the parent to be a USB node now, if I'm not mistaken?

@mike-scott These USB chips can use the I2C bus for management.

https://ww1.microchip.com/downloads/aemDocuments/documents/UNG/ProductDocuments/DataSheets/USB5744-Data-Sheet-DS00001855.pdf

The Kv260 seems to use this way to manage the USB chip.